### ISSUE TITLE
ScheduleForm component bug fixes

### ISSUE THREAD

**Issue by epicasino** [epicasino/mesa-rim-buddy-finder#16]
if nothing is chosen for availability, default values 'from:' and 'to:' are saved to database and shown on explore

**Comment by epicasino** [comment]
Not necessarily an elegant solution, but a solution nonetheless. Work on polishing it later

```
{
     userInfo?.availability?.saturday
            ? userInfo?.availability?.saturday.unavailable
              ? 'Unavailable'
              : userInfo?.availability?.saturday.from !== '' &&
                  userInfo?.availability?.saturday?.to !== ''
              ? `From: ${userInfo?.availability?.saturday?.from} to ${userInfo?.availability?.saturday?.to}`
                : 'N/A'
              : 'N/A'
}
```

**Comment by epicasino** [comment]
resolved in front-end 59e5c85
